src/30_Advanced/ComboBox.html

```diff
@@ -0,0 +1,181 @@
+<!---{
+  "preview": "default"
+  }--->
+<!--
+  ## Introduction
+
+  This sample demonstrastes a classic dropdown combo, with a text field which filters the dropdown elements 
+  according to the user's input. The options are rendered via amp-list.
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <title>ComboBox</title>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/..\30_advanced/combobox/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- ## Setup -->
+  <!--
+  The amp-selector component is used for the dropdown list options. The amp-bind component is used for the filtering of the dropdown options
+  according to the user's input. The fontawesome css icons are imported for the arrow icon.
+  -->
+  <script async custom-element="amp-selector" src="https://cdn.ampproject.org/v0/amp-selector-0.1.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.min.css">
```

Comment:
Would you mind not using font-awesome here? I find it a bit overkill for a single icon (as it quite impacts loading performance). We usually use material icons, e.g. [this](https://material.io/icons/?utm_source=google&utm_medium=cpc&utm_campaign=1001467%20%7C%20Material.IO%20%7C%20Global%20%7C%20en%20%7C%20Hybrid%20%7C%20Text%20%7C%20BKWS&utm_term=%7Bkeyword%7D#ic_keyboard_arrow_down) and [this](https://material.io/icons/?utm_source=google&utm_medium=cpc&utm_campaign=1001467%20%7C%20Material.IO%20%7C%20Global%20%7C%20en%20%7C%20Hybrid%20%7C%20Text%20%7C%20BKWS&utm_term=%7Bkeyword%7D#ic_keyboard_arrow_up) one. You could either copy them to `src/img/*` or inline them via css (what I'd recommend doing).

Reply:
Alright, I don't mind. Removed the font-awesome import and added material icons instead.